app/components/CartComponent.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { PropTypes, Component } from 'react'
+import { Table, Button } from 'react-bootstrap'
+
+export default class CartComponent extends Component {
+  constructor(props) {
+    console.log(props)
+    super(props)
+    this.onUpdateProduct = this.onUpdateProduct.bind(this)
+    this.onRemoveProduct = this.onRemoveProduct.bind(this)
+    this.getSubtotal = this.getSubtotal.bind(this)
+  }
+
+  onUpdateProduct(){
+    console.log("I'm udating!")
```

Comment:
In general you don't want to leave console.logs in the version that goes up to master. You want to think of master as your polished version of the code.